### DISCUSSION TITLE
~10% accuracy difference in lazer between normal and classic mod

### DISCUSSION THREAD

**Discussion by BlackBloodLilly** [ppy/osu#28027] [Q&A] [answered]
I'm currently trying out lazer and I've realised myself missing a lot less using the classic mod compared to the default.
I'm not a good player by any means and this is quite confusing since the game feels more or less the same.
Is this due to accuracy being calculated differently or something chaning I'm not aware off?

I'm just very confused and would like to know why this is happening. I do also have a video of myself playing if someone is interested.

**Comment by TLuzrod** [comment, marked as answer]
Hi!
There are a few gameplay differences between stable and lazer so the accuracy won't be the same depending on the client you use.
The biggest difference when it comes to accuracy is that in lazer sliders count for accuracy.
You can find most changes in the [wiki](https://osu.ppy.sh/wiki/en/Client/Release_stream/Lazer/Gameplay_differences_in_osu!(lazer)#general).
Hope I could help!

**Reply by BlackBloodLilly** [reply]
I think that's it, ty.
